templates/provider/kcm/templates/rbac/controller/roles.yaml

```diff
@@ -5,6 +5,36 @@ metadata:
   labels:
   {{- include "kcm.labels" . | nindent 4 }}
 rules:
+- apiGroups:
```

Comment:
The comment is not related to this particular file but rather to the user permissions: who is supposed to use the new resource? Should we add user-facing roles for it?